src/content/app/browser/browser-region-editor/BrowserRegionEditor.test.tsx

```diff
@@ -265,10 +265,10 @@ describe('<BrowserRegionEditor />', () => {
         userEvent.click(submitButton);
         expect(mockChangeBrowserLocation).toHaveBeenCalled();
 
-        const { ensObjectId, chrLocation } =
+        const { focusId, chrLocation } =
           mockChangeBrowserLocation.mock.calls[0][0];
         const [, start, end] = chrLocation;
-        expect(ensObjectId).toBe(null);
+        expect(focusId).toBe(undefined);
```

Comment:
This suggests that the argument to `changeBrowserLocation` should have a `focusId` field; but the object passed to `changeBrowserLocation` doesn't have such a field. If you want to make sure that you are passing an object without this field, you should probably save the argument that `mockChangeBrowserLocation` was called with and inspect it:

```
const newBrowserLocationData = mockChangeBrowserLocation.mock.calls[0][0];
expect('focusId' in newBrowserLocationData).toBe(false);

or

expect(Object.keys(newBrowserLocationData).includes('focusId')).toBe(false);
```

or, alternatively, don't even check this field if you don't care that it's absent.

Reply:
I've removed this check